packages/build-utils/src/workspaces/workspace-managers.ts

```diff
@@ -6,7 +6,7 @@ import type { Framework } from '@vercel/frameworks';
  * This list is designed to work with the @see {@link detectFramework} function.
  *
  * @example
- *   import { workspaceManagers as frameworkList } from '@vercel/build-utils/workspaces'
+ *   import { workspaceManagers as frameworkList } from '@vercel/build-utils'
```

Comment:
```suggestion
 *   import { workspaceManagers as workspaceManagerList } from '@vercel/build-utils'
```

Reply:
I think we're ok with my changes as we're showing that we can rename `workspaceManagers` to `frameworkList` and pass that into `detectFramework` 🙌